docs/source/notebooks/mmm/mmm_lift_test.ipynb

```diff
@@ -0,0 +1,1716 @@
+{
```

Comment:
<p>Lift tests will be used to alter our model likelihood and  ... ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>good catch. Must of got distracted</p>